packages/app-store/stripepayment/components/index.ts

```diff
@@ -1 +0,0 @@
-export { default as InstallAppButton } from "./InstallAppButton";
```

Comment:
InstallAppButton file doesn't exist. It was there unnecessarily, which means Typechecks are not working on app-store. Need to add that,